include/pmacc/math/operation/SqrtAdd.hpp

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2022 Pawel Ordyna
+ *
+ * This file is part of PMacc.
+ *
+ * PMacc is free software: you can redistribute it and/or modify
+ * it under the terms of either the GNU General Public License or
+ * the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PMacc is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License and the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * and the GNU Lesser General Public License along with PMacc.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "pmacc/types.hpp"
+
+namespace pmacc
+{
+    namespace math
+    {
+        namespace operation
+        {
+            //! Sum square roots
+            struct SqrtAdd
+            {
+                template<typename Dst, typename Src>
+                HDINLINE void operator()(Dst& dst, const Src& src) const
+                {
+                    dst += math::sqrt(src);
```

Comment:
this shoudl be `cupla::math::sqrt`

Sry I missed yesterday that this function in pmacc and not picongpu

Reply:
This also won't always work. I switched now to `DataBoxUnaryTransform` for `sqrt` and removed `AddSqrt`. I hope this will work fine.